libs/api/domains/national-registry-x-road/src/models/nationalRegistryAddress.model.ts

```diff
@@ -1,6 +1,6 @@
 import { Field, ObjectType } from '@nestjs/graphql'
 
-@ObjectType()
+@ObjectType('NationalRegistryXRoadAddress')
```

Comment:
how do we test this to make sure it doesn't break anything?

Reply:
@snaerseljan Can you assist with this, do you know if anyone is querying this directly? The rename was to fix up the schema but I'm worried about breaking something